schema/src/ProjectInfo.yaml

```diff
@@ -55,16 +150,31 @@ classes:
         slot_uri: qudt:unit
         range: string
         description: The unit of measurement for the quantity.
-  Organization:
+  Link:
```

Comment:
Link and Versioned Link can be added to the core file, as they will be used broadly. Also if we do use this we should make sure @clevinson's PR (#35 ) is updated accordingly.